internal/pkg/rerere/rerere.go

```diff
@@ -155,20 +172,26 @@ func Retesting(log *logrus.Entry, ghc githubClient, client gitRepoClient,
 
 // checkContexts checks if all the tests have passed.
 func checkContexts(log *logrus.Entry, ghc githubClient, contexts prowflagutil.Strings,
-	retestingBranch string, org string, repo string) error {
+	retestingBranch string, org string, repo string) (bool, error) {
 	lastCommit, err := ghc.GetSingleCommit(org, repo, retestingBranch)
 	if err != nil {
-		return fmt.Errorf("get %s last commit failed: %v", retestingBranch, err)
+		return false, fmt.Errorf("get %s last commit failed: %v", retestingBranch, err)
 	}
 
 	passedContexts := sets.String{}
 	lastCommitRef := lastCommit.SHA
 	// List all status.
 	statuses, err := ghc.ListStatuses(org, repo, lastCommitRef)
 	if err != nil {
-		return fmt.Errorf("list %s statuses failed: %v", retestingBranch, err)
+		return false, fmt.Errorf("list %s statuses failed: %v", retestingBranch, err)
 	}
+
+	contextsSet := contexts.StringSet()
```

Comment:
Does context here refer to requireContext? Maybe `requireContextSet` will be better?

Reply:
Fixed.